scripts.py

```diff
@@ -1891,11 +1891,15 @@ def collections_with_uploader(self, collections):
             to use for that Collection.
         """
         for collection in collections:
-            uploader = MirrorUploader.for_collection(
+            covers = MirrorUploader.for_collection(
                 collection, ExternalIntegrationLink.COVERS
             )
-            if uploader:
-                policy = self.replacement_policy(uploader)
+            books = MirrorUploader.for_collection(
+                collection, ExternalIntegrationLink.BOOKS
+            )
+            mirror = dict(covers=covers, books=books)
+            if mirror["covers"] or mirror["books"]:
```

Comment:
I'm pretty sure this is stylistic, but i would write `if covers or books:` and build the dictionary inside -- otherwise you're making a dictionary you're not using.